jupyterhub/jupyterhub/base/jupyterhub-cluster-role.yaml

```diff
@@ -22,3 +22,4 @@ rules:
       - ''
     resources:
       - nodes
+      - secrets
```

Comment:
This will give jupyterhub access to all secrets on the cluster. Can we instead use a scoped role + rolebinding instead?